docs/user/api/_index.md

```diff
@@ -220,6 +220,19 @@ Response on success:
 }
 ```
 
+### Accept remote_write requests
+
+```
+POST /agent/api/v1/metrics/instance/{instance}/write
+```
+
+This endpoint accepts Prometheus-compatible remote_write POST requests, and
+appends their contents into an instance's WAL. 
```

Comment:
Can we add a sentence to replace `{instance}` with the instance name from the config? Something like:

``````
Replace `{instance}` with the name of the metrics instance from your config file. For example, this config file defines the instance names "dev" and "prod":

```yaml
metrics:
  configs:
  - name: dev  # /agent/api/v1/metrics/instance/dev/write
  - name: prod # /agent/api/v1/metrics/instance/prod/write
```
``````

Reply:
Yeah, you're right. I've added an extra paragraph, let me know how it looks!